draft-kuehlewind-shmoo-remote-fee.md

```diff
@@ -120,6 +120,23 @@ It is not in scope for this document or the shmoo working group to make suggesti
 the IETF's overall funding model. This is the responsibility of the LLC Board taking agreed principles
 like the one proposed in this document into account.
 
+# Considerations on Use and Misuse of a Free Participation Option
+
+This document does not specify hard any requirements on when to use or not use the free option. The purpose of
+the free option is to enable everybody who is interested in participation to join meetings without the meeting
+fee imposing a financial barrier. These cases cannot be limited to a certain group, like students or "self-founded"
+participants, nor to any specific other restricts, like the number of meeting or previous level of involvement.
+The purpose is simply to maximise participation without barriers in order to make the standards process as open as possible.
+
+It is expected that participants who have financial support to use the regular registration option 
+will do so. Information about which registration payment option was used should not be published on an individual
+basis, however, the ratio of paid vs. free participation may indicate if misuse of the free option occurs in
+future. If the number of paid registrations decreases this can however also have other reasons than misuse,
```

Comment:
We might want to just say that aggregate data should be published and leave it at that, lest people get sidetracked by the idea that the data will be used to draw predetermined conclusions.

Reply:
```suggestion
will do so. Aggregated data on the number of free registrations used should be published as a change in the ratio of paid vs. free participation may indicate if misuse of the free option occurs in
future. If the number of paid registrations decreases this can however also have other reasons than misuse,
```

Like this?

Reply:
I would be even more generic:  "Aggregated data on the number and percentage of free registrations used should be published, as this will permit analysis of the free registration option" or similar.  My worry is that "misuse" is very subjective, and people will interpret that as a veiled threat (c.f. Mozilla talking about "misuse" of the TRR opt-out DNS record).

Reply:
Okay. Done. I added "without revealing personal information." That's okay right?

Reply:
Just realising that I also use the term "misuse" in the section title. Is that okay or do you have a better suggestion?